package.json

```diff
@@ -1,46 +1,47 @@
 {
   "private": true,
-  "packageManager": "yarn@1.22.22",
+  "packageManager": "yarn@4.5.0",
   "repository": "https://github.com/comunica/comunica/",
   "workspaces": [
     "engines/*",
     "packages/*",
     "performance/*"
   ],
   "scripts": {
-    "test-changed": "lerna run test --since HEAD",
-    "build-changed": "lerna run build --since HEAD",
-    "test": "jest",
-    "test:ci": "jest --ci --maxWorkers=4 --coverage",
-    "test:browser": "karma start karma.config.js --single-run",
-    "lint": "eslint . --cache",
-    "lint-fix": "eslint . --fix",
     "build": "yarn run build:ts && yarn run build:components",
+    "build:changed": "lerna run build --since HEAD",
+    "build:components": "componentsjs-generator 'engines/*' 'packages/*'",
+    "build:engines": "lerna run prepare",
+    "build:engines-browser": "lerna run browser",
     "build:ts": "tsc",
-    "build:components": "componentsjs-generator engines/* packages/*",
     "build-watch": "nodemon -e ts --ignore '*.d.ts' --exec yarn run build",
-    "build-watch:ts": "tsc --watch",
     "build-watch:components": "nodemon -e d.ts --exec yarn run build:components",
-    "publish": "yarn run build",
-    "publish-release": "lerna publish",
-    "publish-bare": "lerna exec -- npm publish --silent",
-    "publish-canary": "yarn run build && lerna version prerelease --preid alpha.$(packages/utils-monorepo/sh/get-next-alpha-version.sh @comunica/core) --exact --ignore-scripts --force-publish --no-push --no-git-tag-version --yes && git update-index --assume-unchanged $(git ls-files | tr '\\n' ' ') && lerna publish from-package --no-git-reset --pre-dist-tag next --force-publish --no-push --no-git-tag-version --yes && git update-index --no-assume-unchanged $(git ls-files | tr '\\n' ' ') && git checkout .",
-    "doc": "packages/utils-monorepo/sh/typedoc-json.sh create && typedoc && packages/utils-monorepo/sh/typedoc-json.sh remove",
-    "preinstall": "node -e \"if (!fs.existsSync('packages/runner-cli/bin/run.js')) { var os = fs.createWriteStream('packages/runner-cli/bin/run.js'); os.write('#!/usr/bin/env node\\nconsole.error(\\'Temporary runner script during lerna installation. If you see this, please run yarn install again.\\');\\nprocess.exit(1);'); os.end(); }\" && node -e \"if (!fs.existsSync('packages/runner/bin/compile-config.js')) { var os = fs.createWriteStream('packages/runner/bin/compile-config.js'); os.write('#!/usr/bin/env node\\nconsole.error(\\'Temporary compile-config script during lerna installation. If you see this, please run yarn install again.\\');\\nprocess.exit(1);'); os.end(); }\"",
```

Comment:
Why was this removed?

Reply:
It appears to just print a warning that tells the user to do yarn install again, when the packages in question have not been built. This implicitly relies on the `postinstall` hook to build everything, which is not told to the user anyway. I have never seen the script get called, and the expectation would be that everyone runs build after their changes in a TypeScript project.

So in short, I think it was an unnecessary and unused means to display a message that did not actually say what the problem is, while also relying on the `postinstall` script that I changed to not build everything upon install. I have had situations where I just want an install without doing a build, knowing the build would fail anyway, for example.

Reply:
Please check the git blame history on this.
I don't remember the precise reason, but there is an important reason for this.

Reply:
It seems to have been related to lerna bootstrapping in 83f1feea648760a2739f698b3249a313d3ee1670, but the repo no longer uses lerna bootstrapping as far as I know? It seems to have been something before Yarn workspaces were introduced. The CI also runs fine on Windows without this workaround now (something discussed in #122), so I would imagine it is not needed anymore.

Reply:
Ok, thanks for checking. Looks like this can be removed indeed now.